app/css/cohortbuilder.css

```diff
@@ -223,4 +223,8 @@ a:hover {
 
 #concept-verbose {
     text-indent: '20px';
-}
\ No newline at end of file
+}
+
+.Select.Select--multi .Select-menu-outer {
```

Comment:
Is it possible to make the classname lowercase?

And also if you can reduce the dash to one.

Reply:
I didn't create these classes. I'm just overwriting a rule. I think these classes are from an installed dependency. I'll investigate further

Reply:
Cool. Nice job 👍